src/frontend/apps/desk/src/features/teams/store/useTeamsStore.tsx

```diff
@@ -0,0 +1,19 @@
+import { create } from 'zustand';
+
+import { TeamsOrdering } from '../api/useTeams';
+
+interface TeamsStore {
+  ordering: TeamsOrdering;
+  changeOrdering: () => void;
+}
+
+export const useTeamStore = create<TeamsStore>((set) => ({
+  ordering: TeamsOrdering.BY_CREATED_ON_DESC,
+  changeOrdering: () =>
+    set(({ ordering }) => ({
+      ordering:
+        ordering === TeamsOrdering.BY_CREATED_ON
+          ? TeamsOrdering.BY_CREATED_ON_DESC
+          : TeamsOrdering.BY_CREATED_ON,
+    })),
+}));
```

Comment:
Would it be easier to store a boolean flag `is_descending` or smth? Do you plan to introduce new `TeamsOrdering` values?

Reply:
Let's check with the product team, what they add in mind, if it was date ordering, or name ordering btw.

Reply:
alright agree for the name ordering, please take a look at my first comment !